4.6/Dockerfile

```diff
@@ -60,10 +48,14 @@ RUN set -x \
 	&& sed -ri "s!^(\#\s*)?(elasticsearch\.url:).*!\2 'http://elasticsearch:9200'!" /opt/kibana/config/kibana.yml \
 	&& grep -q 'elasticsearch:9200' /opt/kibana/config/kibana.yml
 
+RUN chmod 666 /opt/kibana/optimize/.babelcache.json
+RUN chmod 777 /opt/kibana/config
```

Comment:
I tested each version - 4.1 didn't seem to need it, 4.6 did, 5 needed similar in a different location. (/opt vs /usr/share).

I ran them with `docker -u=nobody` to simulate what OpenShift/ECS/GCE do to run an unprivileged container.

I'm unsure the limitations of AUFS here so could use some help, I only run on devicemapper or overlay.